dist/gaia-header-es5.js

```diff
@@ -658,10 +658,10 @@
            * Exports
            */
 
-          var base = window.GAIA_ICONS_BASE_URL || window.COMPONENTS_BASE_URL || "bower_components/";
+          var base = window.GAIA_ICONS_BASE_URL || window.COMPONENTS_BASE_URL || 'bower_components/';
 
           if (!document.documentElement) {
-            window.addEventListener("load", load);
+            window.addEventListener('load', load);
```

Comment:
Well, I’m using bower with the defaults of this repository. Haven’t changed anything…

@wilsonpage is there something I miss here?